backend/apis/clients/follow-ups/create-follow-up.api.js

```diff
@@ -79,20 +77,26 @@ app.post("/api/clients/:clientId/follow-ups", (req, res) => {
 
     query += mysql.format(
       `SELECT GROUP_CONCAT(services.serviceName SEPARATOR ', ') services FROM services WHERE id IN (?);`,
-      [serviceIds]
+      [serviceIds.length > 0 ? serviceIds : null]
```

Comment:
`IN (NULL)` is a way of executing the sql statement and guaranteeing it returns nothing